src/openms_gui/source/VISUAL/TVIdentificationViewController.cpp

```diff
@@ -931,7 +931,7 @@ namespace OpenMS
     LayerData& current_layer = current_canvas->getCurrentLayer();
     const SpectrumType& current_spectrum = current_layer.getCurrentSpectrum();
 
-    AASequence aa_sequence = ph.getSequence();
+    const AASequence& aa_sequence = ph.getSequence();
```

Comment:
thread safe?

Reply:
can `ph` be changed by another thread? otherwise I would say yes, it should be